protocol/protos/NetRemoteDataStream.proto

```diff
@@ -23,8 +23,54 @@ message DataStreamUploadData
     bytes Data = 1;
 }
 
+message DataStreamDownloadData
+{
+    bytes Data = 1;
+    uint32 SequenceNumber = 2;
+    DataStreamOperationStatus Status = 3;
+}
+
 message DataStreamUploadResult
 {
     uint32 NumberOfDataBlocksReceived = 1;
     DataStreamOperationStatus Status = 2;
 }
+
+enum DataStreamType
+{
+    DataStreamTypeUnknown = 0;
+    DataStreamTypeFixed = 1;
+    DataStreamTypeContinuous = 2;
+}
+
+enum DataStreamPattern
```

Comment:
Does this refer to the throughput pattern, for example, the timing/rate of the data flow? In that case, `Constant` == an uninterrupted flow of data, correct?

Reply:
Right, `Constant` would be a steady pattern with each data message being the same number of bytes